### DISCUSSION TITLE
Stopping Criteria

### DISCUSSION THREAD

**Discussion by DrAshCox** [asreview/asreview#1483] [Q&A]
Afternoon everyone, I am using ASReview for a SR&MA and have 11,289 to screen after removing duplicates. Based on some of the supporting literature, 95% of these will be irrelevant. So, my question is, would you stop after screening the 5% (565 papers)? If so, what is the supporting citation to justify this for reviewers? This is with the understanding that ASReview will have moved the most relevant to the front of the screening process. My model is below, reference to their justification would also be appreciated. 

Feature extraction: TF-IDF
Classifier: Naive Bayes
Query strategy: Maximum
Balance strategy: Dynamic resampling (double)

Thanks in advance
Ash

**Comment by Rensvandeschoot** [comment]
see also the pre-print "The SAFE Procedure: A Practical Stopping Heuristic for Active Learning-Based Screening in Systematic Reviews and Meta-Analyses": https://psyarxiv.com/c93gq

**Reply by DrAshCox** [reply]
Fantastic, thank you. Is there also a lay breakdown for the model that I can reference for reviewers. We have now included your paper and followed your guidenace for the stopping criteria. Essentially, just a clear breakdown of the alternatives to what I used below. It was one of the reviewer comments.

Feature extraction: TF-IDF
Classifier: Naive Bayes
Query strategy: Maximum
Balance strategy: Dynamic resampling (double)

**Reply by Rensvandeschoot** [reply]
Maybe this paper could be usefull [Performance of active learning models for screening prioritization in systematic reviews: a simulation study into the Average Time to Discover relevant records](https://systematicreviewsjournal.biomedcentral.com/articles/10.1186/s13643-023-02257-7), and this systematic review of all simulation studies conducted comparing different models: [Simulation-based Active Learning for Systematic Reviews: A Systematic Review of the Literature](https://psyarxiv.com/67zmt)
